diff -up kio-5.74.0/src/filewidgets/kfileplacesmodel.cpp.1~ kio-5.74.0/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.74.0/src/filewidgets/kfileplacesmodel.cpp.1~	2020-09-06 12:28:23.000000000 +0200
+++ kio-5.74.0/src/filewidgets/kfileplacesmodel.cpp	2020-09-06 23:12:17.051312615 +0200
@@ -19,6 +19,7 @@
 #include <QMimeData>
 #include <QTimer>
 #include <QFile>
+#include <QStandardPaths>
 #include <QAction>
 #include <QMimeDatabase>
 
@@ -307,6 +308,13 @@ KFilePlacesModel::KFilePlacesModel(const
                 createSystemBookmark(I18NC_NOOP("KFile System Bookmarks", "Downloads"),
                                      QUrl::fromLocalFile(downloadFolder), QStringLiteral("folder-downloads"));
             }
+            // We add Music, Downloads, Pictures, Videos
+            createSystemBookmark(I18NC_NOOP("KFile System Bookmarks", "Music"),
+                                 QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), QStringLiteral("folder-sound"));
+            createSystemBookmark(I18NC_NOOP("KFile System Bookmarks", "Videos"),
+                                 QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), QStringLiteral("folder-video"));
+            createSystemBookmark(I18NC_NOOP("KFile System Bookmarks", "Pictures"),
+                                 QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), QStringLiteral("folder-image"));
 
             createSystemBookmark(I18NC_NOOP("KFile System Bookmarks", "Network"),
                                  QUrl(QStringLiteral("remote:/")), QStringLiteral("folder-network"));
