diff -Naur kio-5.20.0/src/filewidgets/kfileplacesmodel.cpp kio-5.20.0.tpg/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.20.0/src/filewidgets/kfileplacesmodel.cpp	2016-03-05 19:00:04.000000000 +0000
+++ kio-5.20.0.tpg/src/filewidgets/kfileplacesmodel.cpp	2016-03-13 18:25:37.813784858 +0000
@@ -30,6 +30,7 @@
 #include <QtCore/QMimeData>
 #include <QtCore/QTimer>
 #include <QtCore/QFile>
+#include <QtCore/QStandardPaths>
 #include <QColor>
 #include <QAction>
 #include <qmimedatabase.h>
@@ -107,6 +108,22 @@
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               QStringLiteral("Home"), I18N_NOOP2("KFile System Bookmarks", QStringLiteral("Home")),
                                               QUrl::fromLocalFile(QDir::homePath()), QStringLiteral("user-home"));
+        // We add Documents, Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Documents"), ki18nc("KFile System Bookmarks", "Documents").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)), "folder-documents");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Music"), ki18nc("KFile System Bookmarks", "Music").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), "folder-sound");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Downloads"), ki18nc("KFile System Bookmarks", "Downloads").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DownloadLocation) + "/"), "folder-downloads");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Videos"), ki18nc("KFile System Bookmarks", "Videos").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), "folder-video");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Pictures"), ki18nc("KFile System Bookmarks", "Pictures").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), "folder-image");
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               QStringLiteral("Network"), I18N_NOOP2("KFile System Bookmarks", QStringLiteral("Network")),
                                               QUrl(QStringLiteral("remote:/")), QStringLiteral("network-workgroup"));
