diff -up kio-5.65.0/src/filewidgets/kfileplacesmodel.cpp.1~ kio-5.65.0/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.65.0/src/filewidgets/kfileplacesmodel.cpp.1~	2019-12-07 22:44:42.000000000 +0100
+++ kio-5.65.0/src/filewidgets/kfileplacesmodel.cpp	2019-12-09 19:23:31.578881921 +0100
@@ -31,6 +31,7 @@
 #include <QMimeData>
 #include <QTimer>
 #include <QFile>
+#include <QStandardPaths>
 #include <QAction>
 #include <qmimedatabase.h>
 
@@ -303,6 +304,20 @@ KFilePlacesModel::KFilePlacesModel(const
                                                 QUrl::fromLocalFile(downloadFolder), QStringLiteral("folder-downloads"));
         }
 
+        // We add Documents, Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Documents"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)), QStringLiteral("folder-documents"));
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Music"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), QStringLiteral("folder-sound"));
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Videos"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), QStringLiteral("folder-video"));
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Pictures"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), QStringLiteral("folder-image"));
+
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               I18NC_NOOP("KFile System Bookmarks", "Network"),
                                               QUrl(QStringLiteral("remote:/")), QStringLiteral("folder-network"));
