diff -up kio-5.62.0/src/filewidgets/kfileplacesmodel.cpp.1~ kio-5.62.0/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.62.0/src/filewidgets/kfileplacesmodel.cpp.1~	2019-09-07 16:00:19.000000000 +0200
+++ kio-5.62.0/src/filewidgets/kfileplacesmodel.cpp	2019-09-11 21:46:28.954711333 +0200
@@ -34,6 +34,7 @@
 #include <QMimeData>
 #include <QTimer>
 #include <QFile>
+#include <QStandardPaths>
 #include <QAction>
 #include <qmimedatabase.h>
 
@@ -297,6 +298,20 @@ KFilePlacesModel::KFilePlacesModel(const
                                                 QUrl::fromLocalFile(downloadFolder), QStringLiteral("folder-downloads"));
         }
 
+        // We add Documents, Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Documents"), ki18nc("KFile System Bookmarks", "Documents").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)), "folder-documents");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Music"), ki18nc("KFile System Bookmarks", "Music").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), "folder-sound");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Videos"), ki18nc("KFile System Bookmarks", "Videos").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), "folder-video");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Pictures"), ki18nc("KFile System Bookmarks", "Pictures").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), "folder-image");
+
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               QStringLiteral("Network"), I18N_NOOP2("KFile System Bookmarks", "Network"),
                                               QUrl(QStringLiteral("remote:/")), QStringLiteral("folder-network"));
