diff -urN kio-5.9.0/src/filewidgets/kfileplacesmodel.cpp kio-5.9.0-patched/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.9.0/src/filewidgets/kfileplacesmodel.cpp	2015-04-04 22:07:47.000000000 +1000
+++ kio-5.9.0-patched/src/filewidgets/kfileplacesmodel.cpp	2015-04-27 20:14:52.531523947 +1000
@@ -31,6 +31,7 @@
 #include <QtCore/QMimeData>
 #include <QtCore/QTimer>
 #include <QtCore/QFile>
+#include <QtCore/QStandardPaths>
 #include <QColor>
 #include <QAction>
 #include <qmimedatabase.h>
@@ -117,6 +118,22 @@
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               "Home", I18N_NOOP2("KFile System Bookmarks", "Home"),
                                               QUrl::fromLocalFile(QDir::homePath()), "user-home");
+        // We add Documents, Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              "Documents", ki18nc("KFile System Bookmarks", "Documents").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)), "folder-documents");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              "Music", ki18nc("KFile System Bookmarks", "Music").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), "folder-sound");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              "Downloads", ki18nc("KFile System Bookmarks", "Downloads").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DownloadLocation)), "folder-downloads");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              "Videos", ki18nc("KFile System Bookmarks", "Videos").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), "folder-video");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              "Pictures", ki18nc("KFile System Bookmarks", "Pictures").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), "folder-image");
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               "Network", I18N_NOOP2("KFile System Bookmarks", "Network"),
                                               QUrl("remote:/"), "network-workgroup");
