diff -up kio-5.73.0/src/filewidgets/kfileplacesmodel.cpp.1~ kio-5.73.0/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.73.0/src/filewidgets/kfileplacesmodel.cpp.1~	2020-08-01 21:49:16.000000000 +0200
+++ kio-5.73.0/src/filewidgets/kfileplacesmodel.cpp	2020-08-03 00:47:58.263053345 +0200
@@ -31,6 +31,7 @@
 #include <QMimeData>
 #include <QTimer>
 #include <QFile>
+#include <QStandardPaths>
 #include <QAction>
 #include <QMimeDatabase>
 
@@ -303,6 +304,17 @@ KFilePlacesModel::KFilePlacesModel(const
                                                 QUrl::fromLocalFile(downloadFolder), QStringLiteral("folder-downloads"));
         }
 
+        // We add Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Music"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), QStringLiteral("folder-sound"));
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Videos"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), QStringLiteral("folder-video"));
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              I18NC_NOOP("KFile System Bookmarks", "Pictures"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), QStringLiteral("folder-image"));
+
         KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
                                               I18NC_NOOP("KFile System Bookmarks", "Network"),
                                               QUrl(QStringLiteral("remote:/")), QStringLiteral("folder-network"));
