diff -up kio-5.46.0/src/filewidgets/kfileplacesmodel.cpp.0000~ kio-5.46.0/src/filewidgets/kfileplacesmodel.cpp
--- kio-5.46.0/src/filewidgets/kfileplacesmodel.cpp.0000~	2018-05-20 14:46:02.952656062 +0200
+++ kio-5.46.0/src/filewidgets/kfileplacesmodel.cpp	2018-05-20 14:48:20.250448268 +0200
@@ -30,6 +30,7 @@
 #include <QMimeData>
 #include <QTimer>
 #include <QFile>
+#include <QStandardPaths>
 #include <QColor>
 #include <QAction>
 #include <QUrlQuery>
@@ -240,6 +241,21 @@ KFilePlacesModel::KFilePlacesModel(const
                                               QStringLiteral("Home"), I18N_NOOP2("KFile System Bookmarks", "Home"),
                                               QUrl::fromLocalFile(QDir::homePath()), QStringLiteral("user-home"));
 
+
+        // We add Documents, Music, Downloads, Pictures, Videos
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Documents"), ki18nc("KFile System Bookmarks", "Documents").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::DocumentsLocation)), "folder-documents");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Music"), ki18nc("KFile System Bookmarks", "Music").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MusicLocation)), "folder-sound");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Videos"), ki18nc("KFile System Bookmarks", "Videos").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::MoviesLocation)), "folder-video");
+        KFilePlacesItem::createSystemBookmark(d->bookmarkManager,
+                                              QStringLiteral("Pictures"), ki18nc("KFile System Bookmarks", "Pictures").toString("openmandriva-kde-translation"),
+                                              QUrl::fromLocalFile(QStandardPaths::writableLocation(QStandardPaths::PicturesLocation)), "folder-image");
+
         // Some distros may not create various standard XDG folders by default
         // so check for their existance before adding bookmarks for them
         const QString desktopFolder = QStandardPaths::writableLocation(QStandardPaths::DesktopLocation);
